src/Packagist/WebBundle/Entity/Package.php

```diff
@@ -495,6 +500,22 @@ public function getReadme()
         return $this->readme;
     }
 
+    /**
+     * @param array|null $funding
+     */
+    public function setFunding($funding)
+    {
+        $this->funding = $funding ? json_encode($funding) : null;
+    }
+
+    /**
+     * @return array|null
+     */
+    public function getFunding()
+    {
+        return json_decode($this->funding, true);
```

Comment:
this won't work well when `$this->funding` is `null`

Reply:
@stof indeed